client/extensions/woocommerce/woocommerce-services/lib/pdf-label-utils/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
```

Comment:
Other places of Calypso don't use alias for translate or localize. I think that code should be consistent.